terraform/docker/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.7"
 services:
   terraform:
-    image: globalforestwatch/terraform:1.2.2
+    image: globalforestwatch/terraform:v1.4.2
```

Comment:
I see there is a `v` now. This still works okay?

Reply:
This was actually a Bad Decision, and changed back. However that's because it resulted in upgrading the state file to require TF 1.x, not because of the version tag. The tag is just a string, and the "v" is required if the tag was created with it, but needs to be omitted if not.